### PR TITLE
Bug 1940940: Deploy multiple operand replicas

### PR DESCRIPTION
**Pull request by bertinatto**
Currently, we deploy only one replica for both operands (webhook and controller). This apparently leads to the operator going unavailable when draining the node where an operand is running on.

With this patch we run multiple replicas of each operand, i.e., 1 replica in single-node control-planes or 2 replicas in multi-node control-planes.

Also, we set maxSurge=0 and maxUnavailable=1 to make sure nodes are drained one at a time and to prevent more than one pod running on the same node when nodePorts or hostNetwork are used.

CC @openshift/storage

### CODE REVIEW

**Comment by bertinatto** [comment]
/hold cancel

@wking @smarterclayton could you take a look at this approach?

**Comment by bertinatto** [comment]
/hold
until we agree on the details in openshift/library-go/pull/1056.

**Comment by bertinatto** [comment]
/hold cancel
Ready for review.

File: pkg/operator/starter.go [hunk 1]
```diff
@@ -65,8 +69,22 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 
 	versionGetter := status.NewVersionGetter()
 
+	kubeInformersForNamespaces := v1helpers.NewKubeInformersForNamespaces(kubeClient, operatorNamespace, targetNamespace)
```
Comment (jsafrane) [inline pkg/operator/starter.go:72]: You must start the informers below

**Comment by bertinatto** [comment]
Fixed

**Pull request merged**